code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -291,47 +278,76 @@
 				var/mob/living/carbon/C = L
 				switch(firer.a_intent)
 					if(INTENT_HELP)
-						C.visible_message("<span class='danger'>[L] is pulled by [H]'s tentacle!</span>","<span class='userdanger'>A tentacle grabs you and pulls you towards [H]!</span>")
+						C.visible_message("<span class='danger'>[L] is pulled to their feet towards [H]!</span>","<span class='userdanger'>A tentacle grabs you and pulls you up towards [H]!</span>")
 						add_attack_logs(H, L, "[H] pulled [L] towards them with a tentacle")
 						C.throw_at(get_step_towards(H,C), 8, 2)
-						return 1
+						C.AdjustParalysis(-2 SECONDS)
+						C.AdjustStunned(-4 SECONDS)
+						C.AdjustWeakened(-4 SECONDS)
+						C.AdjustKnockDown(-4 SECONDS)
+						C.adjustStaminaLoss(-25)
+						return TRUE
 
 					if(INTENT_DISARM)
 						var/obj/item/I = C.get_active_hand()
 						if(I)
 							if(C.drop_item())
 								C.visible_message("<span class='danger'>[I] is yanked out of [C]'s hand by [src]!</span>","<span class='userdanger'>A tentacle pulls [I] away from you!</span>")
-								add_attack_logs(src, C, "[src] has grabbed [I] out of [C]'s hand with a tentacle")
+								add_attack_logs(H, C, "[src] has grabbed [I] out of [C]'s hand with a tentacle")
```

Comment:
I know it's how these were already written but shouldn't these message read something like:
`"[H] has grabbed [I] out of [C]'s hand with [src]"`

Reading "the tentacle has grabbed something with a tentacle" seems odd.

Reply:
Thought I got all of them

Reply:
Fixed